bin/export-db-to-sqlite.sh

```diff
@@ -7,125 +7,305 @@
+
+# Back up DATABASE_URL, to restore later
+export ORIGINAL_DATABASE_URL=$DATABASE_URL
 
-# 1. Dump out to json from the default, source DB
+# Dump out to JSON from the default, source DB
+echo "Dumping JSON from the source DB:"
 
 python manage.py dumpdata \
     contenttypes \
     --indent 2 \
-    --natural-primary \
-    --natural-foreign \
     --output /tmp/export_contenttypes.json || all_well=false
 
+check_status_and_handle_failure "Dumping contenttypes"
+
 python manage.py dumpdata \
     wagtailcore.Locale \
     --indent 2 \
-    --natural-foreign \
     --output /tmp/export_wagtail_locales.json || all_well=false
 
-
-# Noted exclusions:
-# * Sessions are transient and not needed
-# * ContentTypes are dumped separately
-# * Wagtail Locales are dumped separately
-# * PageSubscription is based on User data, which is not ported over
-# * Renditions are not needed as they can be regenerated
-# * ReferenceIndex is not needed as it can be regenerated
-# * WagtailSearch indices need rebuilding and search history is not important
-
+check_status_and_handle_failure "Dumping wagtailcore.Locale"
+
+# Deliberate exclusions:
+# sessions.Session              # Excluded: security risk
+# contenttypes.ContentType      # Excluded: Dumped separately
+# wagtailusers.UserProfile      # Excluded: PII
+# wagtailimages.Rendition       # Excluded: Renditions
+# wagtail_localize_smartling    # Excluded wholesale: translation data may leak draft content
+# wagtail_localize              # Excluded wholesale: translation data may leak draft content
+# wagtailsearch.IndexEntry      # Excluded: WagtailSearch indices need rebuilding and search history is not important
+# wagtailcore.Locale            # Excluded: dumped separately
+# wagtailcore.ModelLogEntry     # Excluded: may contain PII
+# wagtailcore.CollectionViewRestriction  # Excluded: may include passwords
+# wagtailcore.UploadedFile      # Excluded: unavailable data in local builds
+# wagtailcore.ReferenceIndex    # Excluded: can be rebuilt locally
+# wagtailcore.Revision          # Excluded: drafts may leak pre-published content, or stale/dead content
+# wagtailcore.PageViewRestriction  # Excluded: may include passwords
+# wagtailcore.TaskState         # Excluded: comment field may contain sensitive info
+# wagtailcore.PageLogEntry      # Excluded: may contain sensitive info
+# wagtailcore.Comment           # Excluded: may contain sensitive info
+# wagtailcore.CommentReply      # Excluded: may contain sensitive info
+# wagtailcore.PageSubscription  # Excluded: dependent on User model
+# django_rq.Queue               # Excluded: irrelevant to local use and not a real DB table: data lives in Redis
+
+# Deliberate TEMPORARY INCLUSIONS (because without them we cannot load the data) - tables are
+# cleaned at the end, which is why they are in the tables_to_wipe_after_import variable, defined earlier.
+#
+# auth.User                     # Will be purged because of  PII
```

Comment:
it's not clear where the `auth.User` table is being deleted, I don't see it included in `tables_to_wipe_after_initial_migrate` but I might be missing something

Reply:
Dang, sorry for the incorrect comment - will fix. 

It's in `tables_to_wipe_after_import` - the initial migrations don't set up any default users, but the JSON we extract from the postgres DB contains `auth.User` records. 

The deletion happens down here https://github.com/mozilla/bedrock/pull/14993/files/162e3ecd00c107be3804bb094a215d522be8128f#diff-48cc1a5329ed5e6e3114226c7c28671d2aeeb6b29f1e75324629048597d3f9c7R230-R254